src/main/java/com/users/entities/BaseEntity.java

```diff
@@ -0,0 +1,54 @@
+package com.users.entities;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import org.hibernate.Hibernate;
+
+import java.time.Instant;
+import java.util.Objects;
+
+@Getter
+@Setter
+@MappedSuperclass
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder(toBuilder = true)
+public abstract class BaseEntity {
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @Positive
+    private Long createdAt;
```

Comment:
sterge field urile de create si update si ce tine de ele ca nu sunt cerute in acest task. poti sa stergi si clasa asta si sa pui id ul pentru fiecare entiate si creezi un Base entity cand facem partea de createat si updateat